src/Generators/Webserver/Database/Drivers/MariaDB.php

```diff
@@ -49,7 +49,11 @@ public function created(Created $event, array $config, Connection $connection):
         };
         $grant = function ($connection) use ($config, $createUser) {
             if ($createUser) {
-                return $connection->statement("GRANT ALL ON `{$config['database']}`.* TO `{$config['username']}`@'{$config['host']}'");
+                $privileges = config('tenancy.db.tenant-database-user-privileges', null);
```

Comment:
Instead of using `null` as second argument, use `ALL`, that makes the next line unnecessary.

Reply:
Yes but I wanted to make sure that if the constant is defined to `null` (which is different than being undefined) then everything still works.

What do you think?

Reply:
> if the configuration key cannot be found

yes but what if I have an environment variable set to an empty string in there? The variable will exist and will be defined to null so will it take the default?

I wanted to make sure there is no room for error. If you are sure that `config()` will take the default value *even if the config is defined explicitly to `null` then I'm fine changing my PR :)

Reply:
Okay then rewrite to:

```php
$privileges = config('tenancy.db.tenant-database-user-privileges') ?? 'ALL';
```

It will do an isset, which also checks for empty string.

Reply:
I would be careful with that too, try this:

```
FOO= php -r "var_dump(getenv('FOO') ?? 'ALL');"
```

This prints `string(0) ""`

So if a user do: `'tenant-database-user-privileges' => env('FOO'),` and `FOO=` in its `.env` then it will break I guess.

Reply:
I guess so, but we're not being this strict anywhere else in the code.. It's your own responsibility to set up your environment and it would be pretty easy to debug as well..

Reply:
OK I understand your call!

Reply:
The tests are failing, let's go for the null coalescing operator instead and please try running tests locally before pushing.
```php
$privileges = config('tenancy.db.tenant-database-user-privileges') ?? 'ALL';
```

Reply:
Oh right sorry.